### ISSUE TITLE
feat: allows users to find the source markdown within the rendered html file

### ISSUE THREAD

**Issue by julienreszka** [panglesd/slipshow#53]
This would increase portability and would allow users to edit the presentation themselves.

**Comment by panglesd** [comment]
Yes, that is a great idea and very possible! Thanks for opening the issue.

FTR, markdeep can be a reference for this!
